Applications/WebPortal/WebApp/library/source/pages/account_directory_page/group_selection_box.tsx

```diff
@@ -424,6 +425,10 @@ class SuggestionBox extends React.Component<SuggestionBoxProps> {
     }
   }
 
+  private onChangeIndex( index: number ) {
```

Comment:
Remove this method, the rule is to avoid constructing lambda expressions on every render when it's possible. In this case it's not possible to avoid lambda expressions, so inline the lambda expression directly into the prop on line 392.